lib/Dialect/TritonGPU/IR/LinearLayoutConversions.cpp

```diff
@@ -1,3 +1,8 @@
+#include <optional>
+#include <shared_mutex>
+#include <tuple>
+#include <type_traits>
```

Comment:
Where did you use utilities in `type_traits`?

Reply:
Ah, I thought I needed it for `std::apply` but that's on `<tuple>`. Removing.